index.html

```diff
@@ -68,14 +68,22 @@ <h2>
     </section>
     <section class='bottom-section'>
       <article>
-      	<div> 
-      	  top of card
+      	<div class='top-card'> 
+      	  <img src='Images/star.svg' alt='star-svg' id='js-star' />
+          <img src='Images/delete.svg' alt='delete-svg' id='js-delete' />
```

Comment:
These all look great, my only note is we got dinged last project for alt tags saying -svg since it's an accessible feature, so tags like alt='star icon' would be more readable.

Reply:
I didn't know that, thank you for letting me know. I will update the verbiage and re-push.